.gitignore

```diff
@@ -8,3 +8,4 @@ target/
 /cli/pack/
 /.idea/
 /.idea_modules/
+bin/
```

Comment:
I can't anchor it at the top-level. It's like `target/`. There's one in every subproject.

The IDE-specific gitignores made it into this repo before I was strict enough to reject them.